company/values.md

```diff
@@ -65,12 +89,10 @@ We document our goals and plans publicly so we can engage with customers to buil
 
 Our code and PRs are public, so lack of transparency is only obfuscation, not security. Unless something is immediately exploitable and would put customer data at risk, you should be completely transparent about it, just like any other non-security improvement.
```

Comment:
```suggestion
Our code and PRs are public, so lack of transparency is only obfuscation, not security. Unless something is immediately exploitable and would put customer data at risk, you should be completely transparent about it, just like any other non-security improvement.

Being open and transparent also means sharing your work frequently. You should keep others informed on what you're working on, your progress, and plans. It is helpful in getting more diverse feedback and spotting dependencies between the teams and the solutions. It's your responsibility to make everyone feel welcomed to share their point of view.
```

Reply:
As per [my previous comment](https://github.com/sourcegraph/about/pull/1738#issuecomment-707004515)

Reply:
Addressed in https://github.com/sourcegraph/about/pull/1760